packages/embark-utils/src/index.js

```diff
@@ -1,12 +1,15 @@
+const {canonicalHost, defaultCorsHost, defaultHost, dockerHostSwap, isDocker} = require('./host');
 
-class Utils {
-
-  static joinPath() {
+const Utils = {
+  joinPath: function() {
     const path = require('path');
     return path.join.apply(path.join, arguments);
```

Comment:
One thing I just noticed, didn't catch it in the previous PR (sorry):

```suggestion
     return path.join.apply(path, arguments);
```